jquery.js

```diff
@@ -0,0 +1,3 @@
+$('.dog-item1').hover(function(e) {
+  $('img', this).addClass('tile-border');
```

Comment:
Great! But I'd suggest toggle class so it goes away when the user moves the mouse off of the image.